pixtools/clusters/noise_analysis_SD_kmeans_Clustering.py

```diff
@@ -0,0 +1,165 @@
+# Unlike other noiseplot file (by channel) this file will cluster the SDs, allowing for a mean square analysis
+# If there are distinct clusters able to be seperated by depth, it will indicate that there is a clear relationship to noise
+
+# First import required packages
+import sys
+import json
+
+from turtle import fd
+from channeldepth import *
+from channeldepth import *
```

Comment:
Please could you remove this `turtle` import? It isn't used in this module, and in any case isn't a dependency of pixels so it would be good to keep those limited to what is already needed.

Also what is channeldepth here? I dont see the module